guides/source/autoloading_and_reloading_constants_classic_mode.md

```diff
@@ -1236,13 +1236,13 @@ The code evaluating `Hotel::Image` needs to make sure
 
 In these cases the interpreter issues a warning though:
 
-```
+```ruby
 warning: toplevel constant Image referenced by Hotel::Image
 ```
 
 This surprising constant resolution can be observed with any qualifying class:
 
-```
+```ruby
```

Comment:
Because of the mixed content, I think the highlighting here detracts more than it adds.